### PR TITLE
Update parseFeatureFile to skip lines with gherkin comments.

### PR DESCRIPTION
**Pull request by beersonthewall**
### Summary
Fix for issue #7. Checks for and avoids adding gherkin comments to the final html report. Added a step to validate that no comments are added.

### CODE REVIEW

File: features/generate_report.feature [hunk 1]
```diff
@@ -20,6 +20,7 @@ Feature: Report Generation
           When I give dog food to the dog
           Then the dog will eat it
 
+        # TODO: add examples for left to right and right to left petting directions
```
Comment (jkuester) [inline features/generate_report.feature]: So, probably would be best if this was not a TODO.  At first glace I thought it was a real  TODO and not just an example comment.....

```suggestion
        # Do not need examples for left to right and right to left petting directions
```

Comment (beersonthewall) [reply features/generate_report.feature]: Fixed.

File: src/Generator.js [hunk 2]
```diff
@@ -72,6 +72,9 @@ const getNewPhase = (line) => {
   if (line === '\'\'\'' || line === '"""') {
     return 'DOC_STRING_STARTED';
   }
+  if (line.startsWith('#')){
```
Comment (jkuester) [inline src/Generator.js]: So, the idea behind these phases was that they indicate what _(multi-line) section_ the parser is currently looking at in the feature file. The phase can be important context for understanding future lines.  However, at this point for comments they can be parsed line-by-line and we do not really need to communicate any context for future lines. 

Probably can just do this check down in the parseFeatureFile function like this one 
```.js
} else if (line.startsWith('@')) {
```

Comment (beersonthewall) [reply src/Generator.js]: Changed.

**Pull request merged**